backend/src/data/models/interveiw-note/interview-note.model.ts

```diff
@@ -0,0 +1,41 @@
+import { Model, RelationMappings } from 'objection';
+
+import { DbTableName } from '~/common/enums/enums';
+
+import { Abstract } from '../abstract/abstract.model';
+import { Interview, User } from '../models';
+
+class InterviewNote extends Abstract {
+  public 'note': string;
+
+  public 'interviewId': number;
+
+  public 'authorId': number;
+
+  public static override get relationMappings(): RelationMappings {
+    return {
+      interview: {
+        relation: Model.BelongsToOneRelation,
+        modelClass: Interview,
+        join: {
+          from: `${DbTableName.INTERVIEW_NOTES}.interview_id`,
```

Comment:
Please use camelCase for FK referencing

Reply:
But Vlad asked Alina to swap back to snake_case 😕

Reply:
And after that we discussed the concerns with some broken referencing on relations because of `snakeCaseMapping` and I've made a PR changing all existing FK references to camelCase 😐

Reply:
Oh, seems that I missed it. Thank you!